acceptance/setup/aio/pre-suite/020_AIO_Workarounds.rb

```diff
@@ -0,0 +1,29 @@
+test_name 'temporary aio workarounds: ensure puppet and friends are on the path'
+
+step 'setup the symbolic links that aio packaging is not yet providing'
+puppet_bindir = options[:puppetbindir]
+new_puppet_bindir = '/opt/puppetlabs/bin'
+agents.each do |agent|
+  # stick current puppet hard-path in the ... PATH so we can use configprint
+  agent.add_env_var('PATH', "#{puppet_bindir}/puppet:$PATH")
+  on(agent, "mkdir #{new_puppet_bindir}")
+  on(agent, "ln -s #{puppet_bindir}/puppet " \
+            "#{new_puppet_bindir}/puppet")
+  on(agent, "ln -s #{puppet_bindir}/hiera " \
+            "#{new_puppet_bindir}/hiera")
+  on(agent, "ln -s #{puppet_bindir}/facter " \
+            "#{new_puppet_bindir}/facter")
+  agent.add_env_var('PATH', "#{new_puppet_bindir}:$PATH")
+end
```

Comment:
Alternatively, you can add a delimited path in the node definition, e.g. rhel7.yaml:

```
    puppetbindir: /opt/puppetlabs/hiera/bin:/opt/puppetlabs/puppet/bin
```

Though it's less clear where the workaround is.